<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connection</class>
 <widget class="QDialog" name="Connection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="serialGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>361</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>串口设置</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>30</y>
      <width>111</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="SerialNumCBX"/>
     </item>
     <item>
      <widget class="QComboBox" name="BaudRateCBX"/>
     </item>
     <item>
      <widget class="QComboBox" name="DataBitsCBX"/>
     </item>
     <item>
      <widget class="QComboBox" name="ParityCBX"/>
     </item>
     <item>
      <widget class="QComboBox" name="StopBitCBX"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>158</y>
      <width>45</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>数据位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>209</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>奇偶校验</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>56</y>
      <width>45</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>串口号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>107</y>
      <width>45</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>45</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>109</width>
     <height>259</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancleButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RefreshButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
